website/blog/2021-07-28-release-apache-apisix-2.8-en.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Release Apache APISIX 2.8.0"
+author: Serendipity96
+authorURL: "https://github.com/Serendipity96"
+authorImageURL: "https://avatars.githubusercontent.com/u/23514812?v=4"
+---
+> [@Serendipity96](https://github.com/Serendipity96), Apache APISIX contributor from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+>
+<!--truncate-->
+
+Apache APISIX version 2.8 is released! 🎉 This version has 30+ developers participated, more than 100 PRs had been submitted, and supports **1 new feature, 1 new experience, 2 new plugins, and 2 new ways to develop**. Read and learn about the new features in version 2.8!
+👇👇👇
+
+## Release Notes
+
+### New feature: Independent Keepalive connection pool
+
+Starting with [version 2.7](https://apisix.apache.org/blog/2021/06/29/release-apache-apisix-2.7), we have added Apache APISIX's own patches and the Nginx C module to enhance the native Nginx, allowing dynamic setup of increasing number of Nginx configurations. In the latest release, Apache APISIX already supports the configuration of independent Keepalive connection pools at the Upstream level.
```

Comment:
suggestion: 
1. In the latest release -> in version 2.8
2. already supports --> supports

Reply:
yes, the link is wrong